src/style.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
use * selector only for box-sizing property. Don't use it for margins and paddings